modules/assim.batch/R/pda.define.llik.R

```diff
@@ -0,0 +1,34 @@
+##' Define PDA Likelihood Functions
+##'
+##' @title Define PDA Likelihood Functions
+##' @param all params are the identically named variables in pda.mcmc / pda.emulator
+##'
+##' @return List of likelihood functions, one for each dataset to be assimilated against.
+##'
+##' @author Ryan Kelly
+##' @export
+pda.define.llik.fn <- function(settings) {
+  # Currently just returns a single likelihood, assuming the data are flux NEE.
+  llik.fn <- list()
+  for(i in 1:length(settings$assim.batch$inputs)) {
+    # NEE + heteroskedastic Laplace likelihood
+    if(settings$assim.batch$inputs[[i]]$variable.id == 297 && 
```

Comment:
Given Mike's comment, this is not relevant here but when possible it is better to hard code by name than primary key because using names makes the code easier to understand and less error prone - since the id is an arbitrary integer there is no guarantee that it will be fixed. By contrast, we know that changing a variable name will cause trouble.